samples/snippets/feature_store_service/batch_create_features_sample.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_batch_create_features_sample]
+from google.cloud import aiplatform_v1beta1 as aiplatform
+
+
+def batch_create_features_sample(
+    project: str,
+    featurestore_id: str,
+    entity_type_id: str,
+    location: str = "us-central1",
+    api_endpoint: str = "us-central1-aiplatform.googleapis.com",
+    timeout: int = 300,
+):
+    # The AI Platform services require regional API endpoints.
+    client_options = {"api_endpoint": api_endpoint}
+    # Initialize client that will be used to create and send requests.
+    # This client only needs to be created once, and can be reused for multiple requests.
+    client = aiplatform.FeaturestoreServiceClient(client_options=client_options)
+    parent = f"projects/{project}/locations/{location}/featurestores/{featurestore_id}/entityTypes/{entity_type_id}"
+    requests = [
+        aiplatform.CreateFeatureRequest(
+            feature=aiplatform.Feature(
+                value_type=aiplatform.Feature.ValueType.INT64, description="User age",
+            ),
+            feature_id="age",
+        ),
+        aiplatform.CreateFeatureRequest(
+            feature=aiplatform.Feature(
+                value_type=aiplatform.Feature.ValueType.STRING,
+                description="User gender",
+                monitoring_config=aiplatform.FeaturestoreMonitoringConfig(
```

Comment:
Remove this. We leave monitoring out of GA

Reply:
done